tests/integration/components/live-participants-test.js

```diff
@@ -74,6 +75,25 @@ module('Integration | Component | live-participants', function (hooks) {
       .hasText('No Guests in the stream');
   });
 
+  test('renders host name In the stream ', async function (assert) {
+    this.setProperties({
+      peers: mavenPeerData,
+      profilePic: 'profilepicurl',
+      isKickoutModalOpen: true,
+    });
+    this.set('openKickoutModal', () => {
+      this.isKickoutModalOpen = true;
+    });
+    await render(hbs`<LiveParticipants
+      @user='Hosts'
+      @role='host'
+      @peers={{this.peers}}
+      @minimumParticipants="host"
+      @openKickoutModal={{this.openKickoutModal}}
+    />`);
+    assert.dom('[data-test-sidebar-user="1"]').hasText('Ankush');
```

Comment:
More tests can be added

Reply:
please suggest if you think so

Reply:
Like currently you've only added tests for checking peers prop.
What about the rest of props?
What about the function you've passed?

Reply:
I did not get your comment here, can you specify a little bit because I don't think any scenario is left here to test, but still let me know and if those are not that important then let'e merge this PR